src/test/java/Homework19.java

```diff
@@ -11,9 +11,9 @@ public class Homework19 extends BaseTest {
 
     @Test
     public void DeletePlaylistTest() throws InterruptedException {
-        String playlistName = "test playlist";
+        String playlistName = "My Songs";
 
-        login("demo@class.com", "te$t$tudent");
+        login("serge.uwiduhaye@gmail.com", "te$t$tudent");
         wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//i[@title='Create a new playlist']"))).click();
 
         WebElement newPlaylist = driver.findElement(By.xpath("//li[text()='New Playlist']"));
```

Comment:
You can use "wait.until()" in this line also. It will make your code work more correctly